packages/pds/src/index.ts

```diff
@@ -179,7 +179,7 @@ export class PDS {
     })
 
     server = API(server, ctx)
-    if (ctx.cfg.bskyAppViewEndpoint) {
+    if (ctx.cfg.bskyAppViewEndpoint && ctx.cfg.bskyAppViewDid) {
```

Comment:
I think a case with `bskyAppViewEndpoint` configured but not `bskyAppViewDid` should probably fail at server start if we can swing it easily.  Right now if I'm reading right, it will silently fallback to in-process.

Reply:
yeah I'm down with that 👍

Reply:
ah actually that check (& error throw) is done in the lex server for the proxy 👍  so i'll just delete this out of the if statement